lib/index.js

```diff
@@ -88,26 +17,19 @@ const unicodeNormalization = (s1, s2) => {
 };
 const unicodeNormalizationRule = new FunctionRule(unicodeNormalization);
 
-const RE_WHITESPACE = /[\s]+/g;
+const RE_ANY_WHITESPACE = /[\s]+/g;
 const whitespaceDifferences = (s1, s2) => {
-  return [s1.replace(RE_WHITESPACE, ' '), s2.replace(RE_WHITESPACE, ' ')];
+  return [
+    s1.trim().replace(RE_ANY_WHITESPACE, ' '),
```

Comment:
Match existing behavior (and the name of the rule) by discounting any leading or trailing whitespace.